docs/reference/cli/introduction.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: CLI - Introduction
+---
+
+# Introduction
+
+The Lyrion Music Server provides a command-line interface to the players via TCP/IP. After starting the server, commands and queries may be sent by connecting to a specific TCP/IP port. The server will reply echoing the request (for commands) or by returning the requested data (for queries). By default, the server will listen for connections on TCP/IP port 9090. This format is designed for ease of integration into AMX, Crestron and other automation systems.
+
+See [Using the command-line interface](using-the-cli.md) for details.
+
+
+## Supported Commands
+
+The available commands and queries are listed below, grouped by their scope:
+
+* [General](general.md): general management of the Command Line Interface and of the server.
+* [Players](players.md): management of players.
+* [Database](database.md): mangement of the music database.
+* [Playlist](playlist.md): management of the playlist of each player.
+* [Coumpound Queries](compoundqueries.md): queries to get most of the information about the server or a player in one convenient query, that can be updated by the server automatically.
+* [Notifications](notifications.md): internal server commands echoed to the CLI by using the "listen" or "subscribe" commands.
+* [Alarms](alarms.md): management of alarms.
+
+## Changelog

```

Comment:
I don't think we need the changelog, TBH. Thoughts? We can always remove it at some later stage. But who cares about changes introduced more than a decade ago?...